concise-mid-tag.cddl

```diff
@@ -182,11 +182,12 @@ operational-flags = &(
 )
 
 raw-value-group = (
-  comid.raw-value => raw-value-type
+  comid.raw-value => $raw-value-type-choice
```

Comment:
I am little un-clear of the scenario when the raw value needs to be typed but mask is a common mask (un-typed)? Should not the entire raw-value-group needs to be tagged ?

Reply:
A group is just a substitution into a map. Placing a tag on a group doesn't make sense, you have to place it on the group members.

In the case where Evidence reports a raw value, there isn't a group context as the mask isn't appropriate as Evidence. Hence, the tag should be applied to the raw-value code point.